packages/fluentui/docs/src/examples/components/Toolbar/Usage/ToolbarExampleActionPopupInMenu.shorthand.tsx

```diff
@@ -27,12 +27,23 @@ const CustomPopup = ({ onConfirm }) => {
   );
 };
 
+const toolbarItem = (itemName: string, itemIcon: ShorthandValue<BoxProps>) => {
```

Comment:
I've seen this function about 4 times, we can probably create a commonUtils file in `Toolbar` examples folder and put it there, and re-use.

Reply:
I fully agree and this was my point even when I did changes some time ago, but if I remember correctly someone(not sure if Shift or Marija) was commenting that I should not do it in the reusable way because when we pop-up the example then it will be not available there.
@mnajdova or @miroslavstastny please what do you think can we somehow extract this code to reusable file? please for your recommendation? thank you

Reply:
The examples should serve as an example for the clients on how to do things, we don't want to have there partial code and some utilities that they cannot access

Reply:
Thank you @mnajdova for clarification.